<ui version="4.0" >
 <class>appearanceFrm</class>
 <widget class="QWidget" name="appearanceFrm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="topMargin" >
    <number>26</number>
   </property>
   <item row="0" column="0" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>115</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Appearance:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QComboBox" name="styleCb" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <spacer name="horizontalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>128</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="font" >
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>For the overall look of buttons, menus, etc.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Highlight Color:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QComboBox" name="colorCb" >
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="editable" >
      <bool>false</bool>
     </property>
     <property name="frame" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2" >
    <widget class="QLabel" name="label_4" >
     <property name="font" >
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>For selected text and items</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QCheckBox" name="checkBox_2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Use smooth scrolling</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2" >
    <widget class="QCheckBox" name="minimizeDblClickChk" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string>Minimize when double click on a winow title bar</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>System Font:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="2" >
    <widget class="QComboBox" name="comboBox_3" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="1" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Font Size:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="2" >
    <widget class="QSpinBox" name="spinBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="1" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Text Sample:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2" >
    <widget class="QLineEdit" name="lineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="2" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="4" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>518</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="helpBtn" >
       <property name="minimumSize" >
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="../syspref.qrc" >
         <normaloff>:/icons/tool/question.png</normaloff>:/icons/tool/question.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../syspref.qrc" />
 </resources>
 <connections/>
</ui>
